<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mtrWidgetClass</class>
 <widget class="QMainWindow" name="mtrWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mtrWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_openTp5">
        <property name="text">
         <string>打开tp5文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_startMod">
        <property name="text">
         <string>modtran计算</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_openTp7">
        <property name="text">
         <string>打开tp7文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cRadiance">
        <property name="text">
         <string>辐亮度解析</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cTransmittance">
        <property name="text">
         <string>透过率计算</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clr">
        <property name="text">
         <string>清除文本</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>modtran输出:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="modtranStateText"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mtrWidget.qrc"/>
 </resources>
 <connections/>
</ui>
